### PR TITLE
Release/v2.9.18+918

### PR DESCRIPTION
**Pull request by mihail-varbanov**

### CODE REVIEW

File: assets/strings.en.json [hunk 1]
```diff
@@ -692,6 +694,25 @@
...
+	"panel.health.covid19.pending_family_member.label.text.terms.title": "Terms and Conditions",
+	"panel.health.covid19.pending_family_member.label.text.terms.html": "<p>The person named above has self-identified as a family or household member (“Family Member”) and has requested access to the COVID-19 testing services provided by the University of Illinois (“University”). By touching the “AGREE” button below, you agree that:</p><p>1. You are a University employee currently eligible for COVID-19 testing by the University;<br>2. Family Member is a family member in your household; and<br>3. You are responsible for any unpaid fees incurred by Family Member in obtaining COVID-19 testing services from the University.</p><p>You also agree that you are accepting responsibility on behalf of the Family Member if the Family Member is a minor under age 18 and you are the parent or legal guardian of the Family Member.</p><p>If you believe the person named above is improperly requesting access and/or has improperly obtained your University Identification Number (UIN), please contact [E-MAIL].</p>",
```
Comment (sandeep-ps) [inline assets/strings.en.json:700]:
```suggestion
	"panel.health.covid19.pending_family_member.label.text.terms.html": "<p>The person named above has self-identified as a family or household member (“Family Member”) and has requested access to the COVID-19 testing services provided by the University of Illinois (“University”). By touching the “I Approve” button below, you agree that:</p><p>1. You are a University employee currently eligible for COVID-19 testing by the University;<br>2. Family Member is a family member in your household; and<br>3. You are responsible for any unpaid fees incurred by Family Member in obtaining COVID-19 testing services from the University.</p><p>You also agree that you are accepting responsibility on behalf of the Family Member if the Family Member is a minor under age 18 and you are the parent or legal guardian of the Family Member.</p><p>If you believe the person named above is improperly requesting access and/or has improperly obtained your University Identification Number (UIN), please contact [E-MAIL].</p>",
```

File: assets/strings.es.json [hunk 2]
```diff
@@ -691,6 +693,25 @@
...
+	"panel.health.covid19.pending_family_member.label.text.terms.title": "Terms and Conditions",
+	"panel.health.covid19.pending_family_member.label.text.terms.html": "<p>The person named above has self-identified as a family or household member (“Family Member”) and has requested access to the COVID-19 testing services provided by the University of Illinois (“University”). By touching the “AGREE” button below, you agree that:</p><p>1. You are a University employee currently eligible for COVID-19 testing by the University;<br>2. Family Member is a family member in your household; and<br>3. You are responsible for any unpaid fees incurred by Family Member in obtaining COVID-19 testing services from the University.</p><p>You also agree that you are accepting responsibility on behalf of the Family Member if the Family Member is a minor under age 18 and you are the parent or legal guardian of the Family Member.</p><p>If you believe the person named above is improperly requesting access and/or has improperly obtained your University Identification Number (UIN), please contact [E-MAIL].</p>",
```
Comment (sandeep-ps) [inline assets/strings.es.json:699]:
```suggestion
	"panel.health.covid19.pending_family_member.label.text.terms.html": "<p>The person named above has self-identified as a family or household member (“Family Member”) and has requested access to the COVID-19 testing services provided by the University of Illinois (“University”). By touching the “I Accept” button below, you agree that:</p><p>1. You are a University employee currently eligible for COVID-19 testing by the University;<br>2. Family Member is a family member in your household; and<br>3. You are responsible for any unpaid fees incurred by Family Member in obtaining COVID-19 testing services from the University.</p><p>You also agree that you are accepting responsibility on behalf of the Family Member if the Family Member is a minor under age 18 and you are the parent or legal guardian of the Family Member.</p><p>If you believe the person named above is improperly requesting access and/or has improperly obtained your University Identification Number (UIN), please contact shieldcu@illinois.edu.</p>",
```

File: assets/strings.zh.json [hunk 3]
```diff
@@ -691,6 +693,25 @@
...
+	"panel.health.covid19.pending_family_member.label.text.terms.title": "Terms and Conditions",
+	"panel.health.covid19.pending_family_member.label.text.terms.html": "<p>The person named above has self-identified as a family or household member (“Family Member”) and has requested access to the COVID-19 testing services provided by the University of Illinois (“University”). By touching the “AGREE” button below, you agree that:</p><p>1. You are a University employee currently eligible for COVID-19 testing by the University;<br>2. Family Member is a family member in your household; and<br>3. You are responsible for any unpaid fees incurred by Family Member in obtaining COVID-19 testing services from the University.</p><p>You also agree that you are accepting responsibility on behalf of the Family Member if the Family Member is a minor under age 18 and you are the parent or legal guardian of the Family Member.</p><p>If you believe the person named above is improperly requesting access and/or has improperly obtained your University Identification Number (UIN), please contact [E-MAIL].</p>",
```
Comment (sandeep-ps) [inline assets/strings.zh.json:699]:
```suggestion
	"panel.health.covid19.pending_family_member.label.text.terms.html": "<p>The person named above has self-identified as a family or household member (“Family Member”) and has requested access to the COVID-19 testing services provided by the University of Illinois (“University”). By touching the “I Accept” button below, you agree that:</p><p>1. You are a University employee currently eligible for COVID-19 testing by the University;<br>2. Family Member is a family member in your household; and<br>3. You are responsible for any unpaid fees incurred by Family Member in obtaining COVID-19 testing services from the University.</p><p>You also agree that you are accepting responsibility on behalf of the Family Member if the Family Member is a minor under age 18 and you are the parent or legal guardian of the Family Member.</p><p>If you believe the person named above is improperly requesting access and/or has improperly obtained your University Identification Number (UIN), please contact shieldcu@illinois.edu.</p>",
```

File: lib/ui/settings/SettingsPendingFamilyMemberPanel.dart [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+  }
+
+  Widget _buildProgressIndicator() {
+    return Visibility(visible: _hasProgress, child:
+      CircularProgressIndicator(valueColor: AlwaysStoppedAnimation<Color>(Styles().colors.fillColorSecondary), strokeWidth: 3,));
+  }
+
+  Widget _buildTermsDialog(BuildContext context) {
+    String termsHtml = Localization().getStringEx('panel.health.covid19.pending_family_member.label.text.terms.html', """
+<p>The person named above has self-identified as a family or household member (“Family Member”) and has requested access to the COVID-19 testing services provided by the University of Illinois (“University”). By touching the “AGREE” button below, you agree that:</p>
```
Comment (sandeep-ps) [inline lib/ui/settings/SettingsPendingFamilyMemberPanel.dart:149]:
```suggestion
<p>The person named above has self-identified as a family or household member (“Family Member”) and has requested access to the COVID-19 testing services provided by the University of Illinois (“University”). By touching the “I Accept” button below, you agree that:</p>
```

File: lib/ui/settings/SettingsPendingFamilyMemberPanel.dart [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+<p>
+1. You are a University employee currently eligible for COVID-19 testing by the University;<br>
+2. Family Member is a family member in your household; and<br>
+3. You are responsible for any unpaid fees incurred by Family Member in obtaining COVID-19 testing services from the University.
+</p>
+<p>You also agree that you are accepting responsibility on behalf of the Family Member if the Family Member is a minor under age 18 and you are the parent or legal guardian of the Family Member.</p>
+<p>If you believe the person named above is improperly requesting access and/or has improperly obtained your University Identification Number (UIN), please contact [E-MAIL].</p>""");
```
Comment (sandeep-ps) [inline lib/ui/settings/SettingsPendingFamilyMemberPanel.dart:156]:
```suggestion
<p>If you believe the person named above is improperly requesting access and/or has improperly obtained your University Identification Number (UIN), please contact shieldcu@illinois.edu.</p>""");
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,6 +6,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [2.9.18] - 2021-03-09
+### Added
+- Fire Analytics events in Pending Family Members popup.
+- Added terms and conditions to Pending Family Members popup [#557](https://github.com/rokwire/safer-illinois-app/issues/557).
+
```
Comment (sandeep-ps) [inline CHANGELOG.md:13]:
```suggestion
### Changed
- Minor text changes in Pending Family Members popup.
```

**Comment by sandeep-ps** [comment]
@mihail-varbanov, FYI, I have made some minor text changes in this PR based on JP's request.

**Review by sandeep-ps — APPROVED** [review]

**Comment by mihail-varbanov** [comment]
Thanks @sandeep-ps, everything is perfect.

**Pull request merged**